pkg/v1/mutate/index.go

```diff
@@ -90,6 +95,20 @@ func (i *index) compute() error {
 	}
 	manifest := m.DeepCopy()
 	manifests := manifest.Manifests
+	if len(i.replaces) > 0 {
```

Comment:
This is just unused now, right? We can drop `index.replaces`.

Reply:
We can, but having figured it out how to work it, maybe we should just leave it?

Reply:
OK done.